README.md

```diff
@@ -40,6 +43,22 @@ After you're finished please remove all the comments and instructions!
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
+<<<<<<< HEAD
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverseinc] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverseinc]** is a...
```

Comment:
- [ ] The line `46` is a product of a merge issue. Kindly omit it.
- [ ] Please customize line `56` with the description of your project. the text there is to give you a guideline of what to input.